live-build/misc/ansible-roles/appliance-build.unittest-internal/tasks/main.yml

```diff
@@ -26,6 +26,7 @@
       - python-minimal
       - chromium-browser
       - libxss1
+      - libgtk-3-0
```

Comment:
Can you add a comment in-code explaining that we need to list `libxss1` and `libgtk-3-0` explicitly because it is missing from the dependencies of `chromium-browser`.